packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -730,15 +730,6 @@ export type CallCompositeIcons = {
     StopAllSpotlightMenuButton?: JSX.Element;
 };
 
-// @beta
```

Comment:
Wondering why it is still getting removed - it should be still here right? Since we build on top of api-summary.ts

Reply:
This should be removed as it needs to be in another API file as well. Jose told me there is a special way to make a diff that includes sub exports